lighthouse-core/lib/csp-evaluator.js

```diff
@@ -76,6 +75,18 @@ const UIStrings = {
   /** Message shown when a CSP uses the deprecated disown-opener directive. Shown in a table with a list of other CSP vulnerabilities and suggestions. "CSP" stands for "Content Security Policy". "disown-opener", "CSP3" and "Cross-Origin-Opener-Policy" do not need to be translated. */
   deprecatedDisownOpener: 'disown-opener is deprecated since CSP3. ' +
     'Please, use the Cross-Origin-Opener-Policy header instead.',
+  /**
+   * @description Message shown when a CSP wildcard allows unsafe scripts to be run in the page. Shown in a table with a list of other CSP vulnerabilities and suggestions. "CSP" stands for "Content Security Policy".
+   *  @example {https:} keyword
```

Comment:
Is this the correct example? seems like a plain URL scheme

Is this what we're referring to?
```suggestion
   *  @example {*} keyword
```